### PR TITLE
Some gradle-up code improvements.

### PR DESCRIPTION
**Pull request by MairwunNx**
### Added: 
- Compatibility with gradle wrapper distribution type `all`.

### Changed
- Commit message from `Update gradle wrapper to` to `Gradle wrapper updated to`. _(More correct)_

### Fixed
- Possibility commit other changes when we adding files to commit.

### CODE REVIEW

**Comment by MairwunNx** [comment]
## Review requested @rahulsom. 🟡

File: gradleup.groovy [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
+def lines = wrapperConfig.readLines()
+def distributionType = "bin"
 
 for (int i = 0; i < lines.size(); i++) {
     if (lines[i].startsWith('distributionUrl')) {
-        lines[i] = "distributionUrl=https\\://services.gradle.org/distributions/gradle-${version}-bin.zip".toString()
+        if (lines[i].contains("-all")) {
+            distributionType = "all"
+        }
+        lines[i] = "distributionUrl=https\\://services.gradle.org/distributions/gradle-${version}-${distributionType}.zip".toString()
```
Comment (rahulsom) [inline gradleup.groovy:15]: This is much nicer. Thank you!

File: gradleup.groovy [hunk 2]
```diff
@@ -1,25 +1,29 @@
...
     }
 }
 
-new File("gradle/wrapper/gradle-wrapper.properties").text =
-        lines.join('\n') + '\n'
+wrapperConfig.text = lines.join('\n') + '\n'
 
 new File('/tmp/commit.txt').text =
-        "Update gradle wrapper to ${version}\n\n${api.body}"
+        "Gradle wrapper updated to ${version}\n\n${api.body}"
```
Comment (rahulsom) [inline gradleup.groovy:22]: I can understand why many would find imperative too strong here. But it's there for consistency. Git merge and revert events use imperative. I think it should go back to the imperative for this and PR Json being built below.

**Review by rahulsom — CHANGES_REQUESTED** [review]

File: gradleup.groovy (hunk 2 above)
Comment (MairwunNx) [reply gradleup.groovy:22]: Understand, rollback to `Update gradle wrapper`?

Comment (rahulsom) [reply gradleup.groovy:22]: Yep. Sounds good.

Comment (koppor) [reply gradleup.groovy:22]: Imperative is the current style. See https://github.com/joelparkerhenderson/git_commit_message#summary-keywords and https://chris.beams.io/posts/git-commit/#imperative

Comment (rahulsom) [inline gradleup.groovy:22]:
```suggestion
        "Update gradle wrapper to ${version}\n\n${api.body}"
```

**Pull request merged**